language/move-vm/runtime/src/loader.rs

```diff
@@ -728,21 +747,29 @@ impl Loader {
             Ok(module)
         }
 
-        if let Some(module) = self.module_cache.lock().unwrap().module_at(id) {
-            return Ok(module);
+        {
+            let _timer = LIBRA_MOVEVM_LOAD_MODULE_CACHE_SECONDS.start_timer();
```

Comment:
this is a `HashMap` access under lock, not clear it should measure to anything useful